example/iris

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-java -Xmx2G -cp ../brushfire-scalding/target/scala-2.10/brushfire-scalding-0.6.0-SNAPSHOT-jar-with-dependencies.jar \
+java -Xmx2G -cp ../brushfire-scalding/target/scala-2.10/brushfire-scalding-0.6.4-SNAPSHOT-jar-with-dependencies.jar \
```

Comment:
Yeah, I should probably just revisit these scripts and fix them all to do something a bit more robust (eg not requiring changes to this file when we bump the version).